docs/upgrade_with_cli.md

```diff
@@ -8,7 +8,7 @@ Percona Everest regularly releases updates that contain bug fixes, security impr
 !!! note
      It is recommended to install the latest version of the CLI .
 
-You need to download CLI version >=0.10.0 for the upgrade command to work. You can only upgrade one minor version at a time. For instance, you can upgrade from version 0.9.0 to version 0.10.0.  
+You can only upgrade one minor version at a time. For instance, you can upgrade from version 0.10.0 to version 1.0.0.  
```

Comment:
I suggest that we use a more clear example like:
```suggestion
You can only upgrade one minor version at a time. For instance, you can upgrade from version 1.0.0 to version 1.1.0 but you cannot directly upgrade from version 1.0.0 to version 1.2.0.  
```

I would then add a note/section in the release notes of 1.0.0 saying that even though it's a major version upgrade we actually support upgrading from 0.10.1 to 1.0.0